Makefile

```
@@ -610,13 +610,16 @@ LICENSE_NAME = "Apache License Version 2.0"
 
 DEB_CODENAME := $(shell lsb_release -cs 2> /dev/null)
 DEB_ARCHITECTURE = `dpkg --print-architecture 2>/dev/null`
-DEB_DEPENDENCIES := --depends openssl
+# If we were using native Debian packaging, dpkg-shlibdeps could supply us with
+# accurate dependency information. However, we build packages manually, so we
+# use dependencies of "libssl-dev" as a proxy. Typically this is "libssl1.1".
+DEB_DEPENDENCIES += --depends $(shell apt-cache depends libssl-dev | grep 'Depends:' | cut -d: -f 2-)
```

No functional changes, but the resulting string looks a bit better without leading spaces :)

```suggestion
DEB_DEPENDENCIES += --depends $(shell apt-cache depends libssl-dev | grep 'Depends:' | cut -d: -f 2- | tr -d ' ')
```